src/events/components/CalendarView/CalendarTile.tsx

```diff
@@ -55,9 +55,9 @@ export const CalendarFillerTiles = ({ days }: { days: number[] }) => (
 export const CalendarEvent = (event: IEvent) => (
   <Link {...getEventUrl(event.id)}>
     <a>
-      <HoverCard card={<CalendarHoverCard {...event} />}>
+      <HoverCard card={<CalendarHoverCard eventId={event.id} />}>
         <p className={style.title} style={{ background: getEventColor(event.event_type) }}>
-          {isCompanyEvent(event.event_type, event.company_event) ? event.company_event[0].company.name : event.title}

```

Comment:
..Did we use to show the company name instead of the event title?

Reply:
In some places, yes we did. But it was a very special edge case of 'when there is exactly one company connected to the event' thing